examples/Substructure/JetGrooming.jl

```diff
@@ -0,0 +1,39 @@
+#! /usr/bin/env julia
```

Comment:
One small additional point - favour lowercase for directory names here, so make the directory `substructure`.


Yes, `EDM4hep` breaks this rule, but it's because EDM = Event Data Model, so it's a acronym; plus it's the actual name of the package.